docs/AddNewOp.md

```diff
@@ -2,14 +2,17 @@
 
 Operators are the basic building blocks that define ONNX model. With a rich set of operators, ONNX can describe most DNN and ML models from various frameworks. Improving the process of accepting and adding a new operator to ONNX is crucial for ONNX standard future and its viability.
 
-In this document, we describe the process of accepting a new proposed operator and how to properly submit a new operator as part of ONNX standard. The goal is to improve on what we currently have based on our experience, learning and feedbacks we gathered from the community. 
+In this document, we describe the process of accepting a new proposed operator and how to properly submit a new operator as part of ONNX standard. The goal is to improve on what we currently have based on our experience, learning and feedbacks we gathered from the community.
 
-Adding new operator to ONNX is vital to keep up with latest model architectures and to keep ONNX standard relevant.
+The ONNX specification includes a core set of operators that enable many models. It is a non-goal to add all possible operators, however more operators are added as needed to cover evolving needs.
 
-# Accepting new operator
-The goal of ONNX standard isn't to implement every possible existing operator from all available frameworks. The goal is to cover most models and keep evolving ONNX to cover future models.
+# 4 steps to add an operator
```

Comment:
thanks for adding these. of course, each of these should have a section below with more details :)

Reply:
I thought the detail section cover the detail?

Reply:
Do you want to redo the entire flow?

Reply:
Not really. Let me submit a quick PR to your fork